File: check_site.mjs
import puppeteer from 'puppeteer';

(async () => {
  const browser = await puppeteer.launch();
  const page = await browser.newPage();
  
  page.on('console', msg => {
    if (msg.type() === 'error') {
      console.log('CONSOLE ERROR:', msg.text());
    } else {
      console.log('CONSOLE LOG:', msg.text());
    }
  });

  page.on('pageerror', error => {
    console.log('PAGE ERROR:', error.message);
  });

  page.on('requestfailed', request => {
    console.log('REQUEST FAILED:', request.url(), request.failure().errorText);
  });

  await page.goto('https://medocarezt.vercel.app/', { waitUntil: 'networkidle2' });
  
  const content = await page.content();
  console.log("HTML length:", content.length);
  
  await browser.close();
})();

File: check_site.py
from playwright.sync_api import sync_playwright

def run(playwright):
    browser = playwright.chromium.launch(headless=True)
    page = browser.new_page()

    # Capture console messages
    page.on("console", lambda msg: print(f"CONSOLE [{msg.type}]: {msg.text}".encode('ascii', 'ignore').decode('ascii')))
    
    # Capture page errors
    page.on("pageerror", lambda exc: print(f"PAGE ERROR: {exc}"))

    # Navigate to the Vercel site
    try:
        page.goto("https://medocarezt.vercel.app/admin/login", wait_until="networkidle")
        
        page.fill("input[type='email']", "admin@medocare.com")
        page.fill("input[type='password']", "@Yush_007*@")
        page.click("button[type='submit']")
        
        page.wait_for_timeout(5000)
        toasts = page.locator(".sonner-toast").all_inner_texts()
        print(f"TOASTS: {toasts}")
        print(f"URL: {page.url}")
    except Exception as e:
        print(f"Error navigating: {e}")

    browser.close()

with sync_playwright() as playwright:
    run(playwright)
